modules/installation-dhcp-upi.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-dhcp-upi-{context}"]
```

Comment:
@staebler, does this section sound right to you?

Reply:
The last sentence seems to me to be a toss away. If we are going to mention that the machines can be configured to use static IP addresses, then we should also give them some guidance on how to do that. Otherwise, let's recommend the use of DHCP and leave it at that.

Reply:
I have a follow-up JIRA to add the non-DHCP path later.